frontend/package.json

```diff
@@ -78,6 +78,7 @@
     "react-table": "^7.6.3",
     "react-table-plugins": "^1.3.1",
     "react-tooltip": "^4.2.18",
+    "react-zoom-pan-pinch": "^2.1.3",
```

Comment:
Changes to `frontend/package-lock.json` file also needs to be committed.

Reply:
Not sure if this is related to this, but `npm install` is failing for me:
<img width="789" alt="image" src="https://user-images.githubusercontent.com/20333972/167606254-34969b2e-c8aa-4b7e-b32a-48e08469eea8.png">
@stepinfwd